web/struct_tree_layer_builder.js

```diff
@@ -102,13 +104,13 @@ class StructTreeLayerBuilder {
   #setAttributes(structElement, htmlElement) {
     const { alt, id, lang } = structElement;
     if (alt !== undefined) {
-      htmlElement.setAttribute("aria-label", alt);
+      htmlElement.setAttribute("aria-label", removeNullCharacters(alt));
     }
     if (id !== undefined) {
       htmlElement.setAttribute("aria-owns", id);
     }
     if (lang !== undefined) {
-      htmlElement.setAttribute("lang", lang);
+      htmlElement.setAttribute("lang", removeNullCharacters(lang));
```

Comment:
nit: we should remove all the invisible chars here, the lang value should either be empty or have the format described in this [rfc](https://www.ietf.org/rfc/rfc3066.txt).

Reply:
Sure, that's easily fixed.